src/extension.ts

```diff
@@ -25,24 +25,53 @@ import * as is from 'vscode-languageclient/lib/utils/is';
 
 const CONFIGURATION = RLSConfiguration.loadFromWorkspace();
 
+function getSysroot(env: Object): string | Error {
+    const rustcSysroot = child_process.spawnSync('rustc', ['--print', 'sysroot'], { env });
```

Comment:
Now that I see this, I'm wondering if this should be `rustup run nightly rustc --print sysroot`. I think I saw RLS being called like that, and it'd probably be weird for racer to use another sysroot than RLS.

Do you want me to change this, @nrc?

Reply:
Hmm, yeah, good catch, sounds like we should do that here too

Reply:
Updated to use nighty, and also changed the cargo build call a bit further down (that get triggered when rls_root is set) to explicitly build using nightly, as RL will always be run on nightly IIUC.